fuel-vm/src/tests/test_helpers.rs

```diff
@@ -55,10 +55,19 @@ pub fn run_script(script: Vec<Instruction>) -> Vec<Receipt> {
 }
 
 /// Assert that transaction didn't panic
+#[track_caller]
```

Comment:
Why do we need this?

Reply:
The `track_caller` attribute retargets panic messages to show the caller location by default, which is the desired behavior when using this function.